protowhat/checks/check_files.py

```diff
@@ -50,6 +48,18 @@ def has_dir(state, path, incorrect_msg="Did you create a directory `{}`?"):
 def load_file(relative_path, prefix=""):
     # the prefix can be partialed
     # so it's not needed to copy the common part of paths
-    p = Path(prefix, relative_path)
+    path = Path(prefix, relative_path)
+
+    return get_file_content(path)
+
+
+def get_file_content(path):
+    if not isinstance(path, Path):
+        path = Path(path)
+
+    try:
+        content = path.read_text(encoding="utf-8")
+    except:
+        content = None
```

Comment:
When does this fail? For binary files?

Reply:
If you don't have read access to a binary.